editor/js/Viewport.Controls.js

```diff
@@ -61,6 +61,8 @@ function ViewportControls( editor ) {
 
 	signals.editorCleared.add( function () {
 
+		editor.setViewportCamera( Object.keys( editor.cameras )[ 0 ] );
```

Comment:
Instead of using ` Object.keys( editor.cameras )[ 0 ]`, why not simply `editor.camera`? 

I'm not even sure selecting the first entry in an object gives you the first added property (camera). AFAIK, insertion order is not retained when iterating properties, see https://jsfiddle.net/Lz2c9etp/1/.

Reply:
good point. 

> AFAIK, insertion order is not retained, see https://jsfiddle.net/Lz2c9etp/1/.

Im curios what `Object.keys()`'s order actually is, the ans: https://jsfiddle.net/4wyxzmh1/ :O